### PR TITLE
add Dictionaries (rm clients)

### PR DESCRIPTION
**Pull request by ksvirkou-hubspot**
Tests don't work (need to rewrite)
**PR to feature/v4 (not master)**
https://github.com/HubSpot/hubspot-api-nodejs/issues/76

### CODE REVIEW

**Comment by atanasiuk-hubspot** [comment]
Please link an issue  and provide more description on what is done, what changes are in SDK interface, etc

**Comment by ksvirkou-hubspot** [comment]
https://github.com/HubSpot/hubspot-api-nodejs/issues/76

File: src/discovery/baseDiscovery.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { configuration } from '../configuration'
+
+export class baseDiscovery {
```
[the remark this answers is not in the document]
Comment (ksvirkou-hubspot) [reply src/discovery/baseDiscovery.ts]: Done

**Comment by ksvirkou-hubspot** [comment]
> Since it's a huge update, Please show some examples on current usage, are there any breaking changes?
```
import { hubspot } from './src/hubspot'
import { configuration } from './src/configuration'

async function main() {
    let config = new configuration({apiKey: ''})
    let client = new hubspot(config)
    let contactsResponse = await client.crm.contacts.basicApi.getPage()

    console.log(contactsResponse)
};
main()

```

**Comment by ksvirkou-hubspot** [comment]
Old version
```
const hubspot = require('@hubspot/api-client')

let hubspotClient = new hubspot.Client({ apiKey: process.env.HUBSPOT_API_KEY })
let contactsResponse = await hubspotClient.crm.contacts.basicApi.getPage()
```
new version
```
const hubspot = require('@hubspot/api-client')

let config = new hubspot.configuration({apiKey: ''})
let client = new hubspot.Hubspot(config)
let contactsResponse = await client.crm.contacts.basicApi.getPage()
```

**Pull request closed without merging**